spec/event_tracer/dynamo_db/worker_spec.rb

```diff
@@ -1,6 +1,13 @@
 require 'spec_helper'
 
 describe EventTracer::DynamoDB::Worker do
+  before do
+    EventTracer::Config.configure do |config|
+      config.app_name = 'test_app'
+      config.dynamo_db_table_name = 'test_table'
+    end
+  end
```

Comment:
Do we need ` EventTracer::Config.reset_config` in after block?

Reply:
Hmmm yes we do. Tried without and other specs fail.

Reply:
Should we add that in this test?

Reply:
Ah, so that we don't pollute the global config space right? Good idea.